op-node/node/config.go

```diff
@@ -138,6 +138,18 @@ func (cfg *Config) Check() error {
 		if err := cfg.Beacon.Check(); err != nil {
 			return fmt.Errorf("misconfigured L1 Beacon API endpoint: %w", err)
 		}
+		if cfg.Rollup.CanyonTime == nil {
```

Comment:
This all should go inside of `cfg.Rollup.Check()` so that everwhere that calls `Check` gets this

Reply:
Good point, done!